README.md

```diff
@@ -10,39 +10,44 @@
   </a>
 </p>
 
-![Dev Protocol](https://user-images.githubusercontent.com/73097560/126608267-c422adf1-d201-4ef4-ad40-502dc69c4f6f.png)
+![Banner](https://user-images.githubusercontent.com/73097560/126608267-c422adf1-d201-4ef4-ad40-502dc69c4f6f.png)
+
+# 💬 Community Discussion
 
-# Community Discussion
 This repository is dedicated to our community discussion, we are talking about a lot of things here like incoming events, new proposals, new features, and many more. You are free to join in this discussion, help us to improve our project, suggest your ideas and give your opinion freely.
 
-## Why you should join this community?
+## 🤔 Why you should join this community?
+
 - Accelerate your learning by contributing with our dApps open-source
 - Get interact more with Web3 and Blockchain
 - Interact with great members
 
-[Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)
+### [Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)
```

Comment:
```suggestion
**[Join now](https://github.com/dev-protocol/community/issues/new?assignees=&labels=github-invitation&template=invitation.yml&title=Please+invite+me+to+the+community) | [Explore](https://github.com/dev-protocol)**
```

Reply:
IMO, it looks better using `###` as it makes the letters bigger.